### PR TITLE
ci: use shared workflows

### PR DESCRIPTION
**Pull request by TheUnderScorer**

### CODE REVIEW

File: .github/workflows/docs-and-coverage.yml [hunk 1]
```diff
@@ -6,32 +6,6 @@ on:
...
-
-      - name: Prepare folder for gh-pages
-        run: mv coverage/lcov-report ./docs/coverage
-
-      - name: Deploy 🚀
-        uses: JamesIves/github-pages-deploy-action@8817a56e5bfec6e2b08345c81f4d422db53a2cdc
-        with:
-          branch: gh-pages
-          folder: docs
+  generate-docs-and-coverage:
```
Comment (ilfa) [inline .github/workflows/docs-and-coverage.yml:9]: Looks like we need to configure paths

Comment (ilfa) [reply .github/workflows/docs-and-coverage.yml:9]:
```yml
with:
      prepare-gh-pages-commands: |
        mv docs/* ./gh-pages
        mv coverage/lcov-report ./gh-pages/coverage
```

Comment (TheUnderScorer) [reply .github/workflows/docs-and-coverage.yml:9]: But it's the [default value](https://github.com/fingerprintjs/dx-team-toolkit/blob/main/.github/workflows/docs-and-coverage.yml#L7) for this input, right?

**Review by ilfa — APPROVED** [review]

**Pull request merged**